client/src/components/Dropdowns/InterestsSelect/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC, ReactElement } from 'react';
+import { MultiValue, ActionMeta } from 'react-select';
+
+import MultiSelect from '../MultiSelect';
+import { useInterests } from '../../../contexts';
+
+interface InterestsSelectProps {
+  onChange: (
+    newValue: MultiValue<{
+      value: string;
+      label: string;
+    }>,
+    actionMeta: ActionMeta<{
+      value: string;
+      label: string;
+    }>,
+  ) => void;
+  values: string[];
+}
+
+const InterestsSelect: FC<InterestsSelectProps> = ({
```

Comment:
do you think we should make one of these for teams as well?

Reply:
Yea probably.